app/uk/gov/hmrc/gform/submission/destinations/DataStoreSubmitter.scala

```diff
@@ -104,6 +109,35 @@ class DataStoreSubmitter(
     DataStoreFileGenerator(userSession, dataStoreMetaData, payloads, dataStore.includeSessionInfo)
   }
 
+  override def validateSchema(
+    dataStore: DataStore,
+    payload: String,
+    withPureJson: Boolean
+  ): String =
```

Comment:
`withPureJson` seems odd. Can we make this method to return `Boolean` and let calling side decide what to do on error?